recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: behst
+  version: 0.4
+
+source:
+  fn: a2a25d973909.zip
+  url: https://bitbucket.org/hoffmanlab/behst/get/a2a25d973909.zip
+  sha256: cf4cc7179e6311f72d350c4fd408930b367b568f4ca77469474c4e5044ffd015
+
+build:
+  number: 1
+  skip: True # [not py27 or osx]
+  
+requirements:
+  build:
+    - python
+    - pandas
+    - zlib
+    - libpng
+    - bedtools
+    - pybedtools
+    - r-base
+    - curl
+    - r-gprofiler
+    - gnu-wget
+  run:
+    - python
+    - pandas
+    - zlib
+    - libpng
+    - bedtools
+    - pybedtools
+    - r-base
+    - curl
+    - r-gprofiler
+    - gnu-wget
+    
+test:
+  files:
```

Comment:
you haven't really included a test ...

You should add something like `project.sh --help`